Content.Shared/Overlays/Switchable/NightVisionComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared.Actions;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Overlays.Switchable;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class NightVisionComponent : SwitchableOverlayComponent
+{
+    public override string? ToggleAction { get; set; } = "ToggleNightVision";
+
+    public override Color Color { get; set; } = Color.FromHex("#98FB98");
```

Comment:
```suggestion

    [DataField]
    public override Color Color { get; set; } = Color.FromHex("#98FB98");
```
I want to be able to make nightvision sources that have different colors, so this should be a datafield. :)

Reply:
NightVisionComponent inherits the Color variable from BaseOverlayComponent; there's no need to use DataField. It works fine without it, as can be verified by looking at the prototypes.

Reply:
Yea, you're right about this. I apoliogize, I was reviewing the code on my phone while in a college bathroom lol.